scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1505,7 +1507,10 @@ private void resumeExistingSession(final ClientHello clientHello, final Record r
 		}
 		if (ticket != null) {
 			// session has been found in cache, resume it
-			Connection peerConnection = getInitialConnection(connection, false);
+			final Connection peerConnection = getInitialConnection(connection, false);
+			if (peerConnection == null) {
```

Comment:
When this should happened ? should we log it ?

Reply:
When the previous is removed, and the new is added, but the random cid doesn't work.
The log will be added.